controlpanel/api/cluster.py

```diff
@@ -23,6 +23,16 @@
 TOOL_RESTARTING = "Restarting"
 TOOL_UPGRADED = "Upgraded"
 TOOL_STATUS_UNKNOWN = "Unknown"
+HOME_RESETTING = "Resetting"
+HOME_RESET_FAILED = "Failed"
+HOME_RESET = "Reset"
```

Comment:
As I said in other comment I'd be explicit and call this "Reset Home" etc everywhere to avoid any possible ambiguity with other related or not operations.